bin/opds_import_monitor

```diff
@@ -6,5 +6,10 @@ import sys
 bin_dir = os.path.split(__file__)[0]
 package_dir = os.path.join(bin_dir, "..")
 sys.path.append(os.path.abspath(package_dir))
+
 from core.scripts import OPDSImportScript
+
+# NOTE: We need to import it explicitly to initialize MirrorUploader.IMPLEMENTATION_REGISTRY
+from core import s3
```

Comment:
Re: import order - Can this be imported before`from core.scripts import OPDSImportScript`?